src/stylesheets/event.css

```diff
@@ -0,0 +1,11 @@
+.alphabetCircle {
```

Comment:
@toshitapandey You can add this to the style block of item.vue. HERE 
https://github.com/geekyd/Event-Buddy/blob/215d2592fdb72268e51459bdc654f46f846fbb1e/src/components/item.vue#L36